lib/src/common/github.dart

```diff
@@ -352,7 +352,13 @@ class GitHub {
       final json = jsonDecode(response.body);
       message = json['message'];
       if (json['errors'] != null) {
-        errors = List<Map<String, String>>.from(json['errors']);
+        try {
+          errors = List<Map<String, String>>.from(json['errors']);
+        } catch (_) {
+          errors = [
```

Comment:
The error format from this API doesn't match others, so handle this one differently